modules/tinymce/src/core/test/ts/browser/paste/InternalClipboardTest.ts

```diff
@@ -17,26 +17,29 @@ describe('browser.tinymce.core.paste.InternalClipboardTest', () => {
   const dataTransfer = Singleton.value<DataTransfer>();
   const lastPreProcessEvent = Singleton.value<EditorEvent<PastePreProcessEvent>>();
   const lastPostProcessEvent = Singleton.value<EditorEvent<PastePostProcessEvent>>();
+  const lastBeforeInputEvent = Singleton.value<EditorEvent<InputEvent>>();
   const lastInputEvent = Singleton.value<EditorEvent<InputEvent>>();
+  let eventTypes: string[] = [];
+  const setEventSingletonAndAddType = (singleton: Singleton.Value<EditorEvent<any>>, event: EditorEvent<any>) => {
```

Comment:
Isn't the any here T since they should be the same. You should be able to feed a `EditorEvent<T>` to a `Singleton.Value<EditorEvent<U>>`.

Reply:
Thanks for the suggestion! Will use generics over `any` where possible next time.